tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Time Series Modeling
+description: This lab will go over the fundamentals of Time Series Modeling, including decomposition, autocorrelation and modeling historical changes.
+author: ghidora77, benamie
+tags: remote-sensing, time-series, satellite
+date_published: 2022-02-01
+---
+<!--
+Copyright 2021 The Google Earth Engine Community Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Lab 05 - Time Series Modeling
+
+## Overview
+
+The purpose of this lab is to establish a foundation for time series analysis on remotely sensed data. You will be introduced to the fundamentals of time series modeling, including decomposition, autocorrelation and modeling historical changes. At the completion of this lab, you will be able to build an explanatory model for temporal data which can be used in many different avenues of research.
+
+This lab is a segment of the [remotesensing.dev](https://www.remotesensing.dev/docs/intro), designed as six-part series on using Google Earth Engine for research in Social Science and economics. 
```

Comment:
Regarding "lab"
Maybe switch from "lab" to "tutorial" throughout for consistency with our other content, which is generally described as tutorials.

Reply:
Grammar fixes, please use:

`This tutorial is a segment of the [remotesensing.dev](https://www.remotesensing.dev/docs/intro), a six-part series on using Google Earth Engine for research in social science and economics.`

Reply:
I switched out all references of 'lab' to 'tutorial'